dashboard/src/modules/components/OverviewComponent/common-component.jsx

```diff
@@ -187,13 +188,14 @@ export const EditRow = (props) => {
         <div>
           <Button
             isDisabled={
-              !props.item[CONSTANTS.IS_DIRTY] ||
+              !(
+                props.item[CONSTANTS.IS_DIRTY_NAME] ||
+                props.item[CONSTANTS.IS_DIRTY_SERVER_DELETE]
+              ) ||
```

Comment:
Here's a place where we're looking at all the "dirty" flags at once.  If they were grouped in a sub-object of `item`, then we could iterate over them all without referencing any of them individually.  (And, we could reset them in a similar way.)

At this point, I'm not suggesting that you change back to having them as a group.  I'm just saying that I didn't see this code when I indicated that there wasn't any point in grouping them.

I do suggest that you consider refactoring this code into a function call.  This code only needs to know whether _something_ is "dirty", and it doesn't need to be concerned with the details of how that determination is made; and, if you defer that determination to a function, you can more easily change it later (e.g., to group the flags and iterate over them).